sigpy/plot.py

```diff
@@ -100,6 +101,7 @@ def __init__(
         self.title = title
         self.interpolation = interpolation
         self.mode = mode
+        self.colormap=colormap
```

Comment:
Hi @Zhitao-Li , sorry for the late reply. Can you add spaces around the equal sign? This is to conform with [PEP8](https://www.python.org/dev/peps/pep-0008/)